frontend/src/components/RunInfo/RunInfoSection.tsx

```diff
@@ -50,7 +50,7 @@ const RunInfoSection = ({ runInfo }: { runInfo: RunInfo }) => {
                     return (
                       <tr key={label}>
                         <td>{label}</td>
-                        <td>
+                        <td style={{ maxWidth: "200px" }}>
```

Comment:
I'm not sure pixels are the right unit to use here. Do you know that they are?

Reply:
yes, I researched on it

Reply:
you can try it once in the browser, by minimizing and maximinzing the window

Reply:
okay, I understand , what I did in my latest commit was to make it more generic, otherwise pixels also work ,
Pixels are a more absolute way of defining, what I now did in my latest commit is a reletive way